lib/dnsimple/client/registrar.rb

```diff
@@ -41,6 +41,27 @@ def register(account_id, domain_name, attributes = {}, options = {})
         Dnsimple::Response.new(response, Struct::Domain.new(response["data"]))
       end
 
+      # Renews a domain.
+      #
+      # @see https://developer.dnsimple.com/v2/registrar/#renew
+      #
+      # @example Renew example.com for 3 years:
+      #   client.registrar.renew(1010, "example.com", period: 3)
+      #
+      # @param  [Fixnum] account_id the account ID
+      # @param  [#to_s] domain_name The domain name to register.
+      # @param  [Hash] attributes
```

Comment:
I think we should make more clear (possibly even in the other method) what is the different between attributes and options. We should at least explain that

````
@param  attributes [Hash] The renewal payload that will be sent in the request body.
```

PS. We should also update our docs to use the correct YARD syntax

```
@param  attributes [Hash] bla bla bla
```

instead of

```
@param  [Hash] attributes bla bla bla
```

It seems YARD understands it, but it may cause troubles. We should switch to the correct one, may be as a a final cleanup.